src/Rector/v8/v0/RemoveCharsetConverterParametersRector.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Ssch\TYPO3Rector\Rector\v8\v0;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\MethodCall;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+use TYPO3\CMS\Core\Charset\CharsetConverter;
+
+/**
+ * @see https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/8.0/Breaking-74031-CharsetConverterParametersRemoved.html
+ */
+final class RemoveCharsetConverterParametersRector extends AbstractRector
+{
+    /**
+     * @return string[]
+     */
+    public function getNodeTypes(): array
+    {
+        return [MethodCall::class];
+    }
+
+    /**
+     * @param MethodCall $node
+     */
+    public function refactor(Node $node): ?Node
+    {
+        if (! $this->isMethodStaticCallOrClassMethodObjectType($node, CharsetConverter::class)) {
+            return null;
+        }
+
+        if (! $this->isNames($node->name, ['entities_to_utf8', 'utf8_to_numberarray'])) {
+            return null;
+        }
+
+        if (count($node->args) <= 1) {
+            return null;
+        }
+
+        if (3 === count($node->args)) {
```

Comment:
I didn't see a better way to do this, if checking the function-name I would have to check the arguments count anyway to avoid the risk of undefined index.

Reply:
@tomasnorre Just by reading i think it should be enough to set $node->args = [$node->args[0]];
 
It is not necessary to remove the arguments explicitly. But you should return $node and not null. Because you modified it and subsequent rectors should be informed by it.